CHANGELOG.md

```diff
@@ -51,6 +51,8 @@
 * Location tracking and routing resources are now freed if no `RouteController` or `PassiveLocationManager` instance is actively being used. ([#3724](https://github.com/mapbox/mapbox-navigation-ios/pull/3724))
 * Added an optional `datasetProfileIdentifier` argument to the `MapboxRoutingProvider(_:settings:datasetProfileIdentifier:)`, `PassiveLocationManager(directions:systemLocationManager:eventsManagerType:userInfo:datasetProfileIdentifier:),` `TilesetDescriptorFactory.getSpecificVersion(version:completionQueue:datasetProfileIdentifier:completion:)`, and `TilesetDescriptorFactory.getLatest(completionQueue:datasetProfileIdentifier:completion:)` methods for obtaining routing tiles optimized for a particular mode of transportation. Make sure to configure `MapboxRoutingProvider` and `TilesetDescriptorFactory` with the correct dataset profile if you customize `Directions.profileIdentifier`. ([#3717](https://github.com/mapbox/mapbox-navigation-ios/pull/3717))
 * Fixed an issue where release builds would crash on invalid navigator initialization. ([#3738](https://github.com/mapbox/mapbox-navigation-ios/pull/3738))
+* Added `routeShieldRepresentationKey` to the user info dictionary of `Notification.Name.passiveLocationManagerDidUpdate` posted by `PassiveLocationManager`, and the `Notification.Name.currentRoadNameDidChange` posted by `RouteController`. The corresponding value is a `MapboxDirections.VisualInstruction.Component.ImageRepresentation` object representing the road shield the user is currently traveling on. ([#3723](https://github.com/mapbox/mapbox-navigation-ios/pull/3723))
+* Removed `NavigationMapView.labelCurrentRoadFeature(at:router:wayNameView:roadNameFromStatus:)`. Instead, updating the road label through the `WayNameView.label.updateRoad(roadName:representation:)` and `WayNameView.label.updateStyle(styleURI:)`. ([#3723](https://github.com/mapbox/mapbox-navigation-ios/pull/3723))
```

Comment:
This is an internal method, so we don’t need to mention it in the changelog. However, we should mention that route shields in the instruction banner are now visually consistent with route shields on the map and in the current road name label.

Reply:
Thanks for the suggestion. Because this PR doesn't include the instruction banner replacement. And the road shield in the road name label used to use map query, which already is consistent with the map. I think this changeLog entry should be left for the instruction banner update one.